src/components/30-organisms/table-sortable/index.js

```diff
@@ -0,0 +1,202 @@
+import { LitElement, css, unsafeCSS, html } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import defineOnce from '../../../utils/define-once';
+import tableCss from './index.scss';
+// TODO use THE NPM package one axa-table is released
+import '../table';
+
+const ASC = 'ascending';
+const DESC = 'descending';
+
+const mapAsc = {
+  [ASC]: 'ASC',
+  [DESC]: 'DESC',
+};
+
+class AXATableSortable extends LitElement {
+  constructor() {
+    super();
+    this.defaultModel = {
+      thead: [],
+      tbody: [[]],
+      tfoot: [[]],
+    };
+    this.model = { ...this.defaultModel };
+    this.innerscroll = 0;
+    this.firstRender = true;
+    this.numCollator = new Intl.Collator(undefined, {
+      numeric: true,
+      sensitivity: 'base',
+    });
+    this.strCollator = new Intl.Collator(undefined, {
+      sensitivity: 'variant',
+    });
+    this.lastIndex = -1;
+  }
+
+  static get tagName() {
+    return 'axa-table-sortable';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(tableCss)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      model: { type: Object },
+      innerscroll: { type: Number },
+    };
+  }
+
+  validateModel() {
+    const {
+      thead: { length: theadL },
+      tbody: {
+        0: { length: tbodyL },
+      },
+      tfoot: {
+        0: { length: tfootL },
+      },
+    } = this.model;
+
+    return !(theadL !== tbodyL && (!tfootL || tfootL === tbodyL));
```

Comment:
What does the L's stand for? tbodyL?

Reply:
probably for lenght. I suggest to not use abbreviations

Reply:
length, i will rename it

Reply:
i cant as prettier now makes it even less readable:

```
return !(
      theadLength !== tbodyLength &&
      (!tfootLength || tfootLength === tbodyLength)
    );
```

i leave it